pulsar-transformations/pom.xml

```
@@ -72,6 +72,21 @@
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>${pulsar.groupId}</groupId>
```

```suggestion
      <groupId>com.datastax.oss</groupId>
```
This is not from Pulsar core. It's a datastax only project so we won't switch between forks.